### PR TITLE
add per grant prize badge for projects

### PR DESCRIPTION
**Pull request by F-Shahali**
#### Reference Issues/PRs
#14
#### What does this implement/fix? Explain your changes.
- grant prize badge added for projects.

#### Any other comments?
This is the result:
Screenshot from 2024-06-23 01-16-19
Screenshot from 2024-06-23 01-16-36

### CODE REVIEW

File: src/Views/Pages/Home/Sections/Section3/index.tsx [hunk 1]
```diff
@@ -270,6 +274,7 @@ const projects = [
 		github: 'https://github.com/sepandhaghighi/pycm',
 		forkPage: 'https://github.com/sepandhaghighi/pycm/forks',
 		starsPage: 'https://github.com/sepandhaghighi/pycm/stargazers',
+		grant: ['PSF Development Grant 2022', 'Nlnet Grant 2022', 'Trelis 2024'],
```
Comment (AHReccese) [inline src/Views/Pages/Home/Sections/Section3/index.tsx:277]: Trelis AI Grant 2024

File: src/Views/Pages/Home/Sections/Section3/index.tsx [hunk 2]
```diff
@@ -327,6 +334,7 @@ const projects = [
 		dark_logo: artDark,
 		light_logo: artLight,
 		account: 'sepandhaghighi',
+		grant: ['PSF 2024'],
```
Comment (AHReccese) [inline src/Views/Pages/Home/Sections/Section3/index.tsx:337]: PSF Grant 2024

**Review by AHReccese — CHANGES_REQUESTED** [review]
Thank you for your efforts, Take a look at my comments.

**Review by AHReccese — COMMENTED** [review]
Thank you for your efforts.
- use `NLnet` instead of Nlnet
- use this structure for PSF grants: PSF Development Grant YEAR
@F-Shahali

**Comment by AHReccese** [comment]
@F-Shahali 

Also in the mobile mode, show the description after clicking on the badge.
On the desktop mode, it does show the description after hovering the badge, but I think it's better to show it after clicking(and not hovering) too.

In the mobile mode, we should handle the touch event instead, as a way of clicking.

**Comment by AHReccese** [comment]
@F-Shahali 
There is an urgent issue which I'm going to address after this PR, it would be great if you could handle the feedback ASAP.

Thanks

**Comment by F-Shahali** [comment]
> @F-Shahali There is an urgent issue which I'm going to address after this PR, it would be great if you could handle the feedback ASAP.
> 
> Thanks

Sure, I will do this within the next one hour.

**Comment by F-Shahali** [comment]
> @F-Shahali
> 
> Also in the mobile mode, show the description after clicking on the badge. On the desktop mode, it does show the description after hovering the badge, but I think it's better to show it after clicking(and not hovering) too.
> 
> In the mobile mode, we should handle the touch event instead, as a way of clicking.

There is a weird problem with showing the description by clicking in desktop mode. I handled it by hovering and it works now for mobile mode after clicking on the badge.
If you agree, we can have it in this way and after addressing the urgent issue, I will come back to resolve the clicking problem.

**Review by AHReccese — APPROVED** [review]
Thanks for your effort
I agree
LGTM

**Pull request merged**